VnE/Source/Extensions/ChemPack/contacts.py

```
@@ -590,10 +584,12 @@ def recurse(node: Node, sub_node: Node, memory=None, result=None):
                         else:
                             sub_node_copy = sub_node_copy_init.__deepcopy__()
                             break
-        if all([x.assigned_node for x in sub_node_copy.connect]):
-            return True, sub_node_copy, result
-        else:
-            return False, sub_node_copy, result
+                probs_nodes.append(sub_node_copy)
+        for prob_node in probs_nodes:
+            if all([x.assigned_node for x in prob_node.connect]):
```

Ruff (0.15.2)

[convention] Unnecessary list comprehension

Remove unnecessary comprehension

(C419)